ops/k8s/deployments/oidc-proxy-deployment.yaml

```diff
@@ -14,9 +14,24 @@ spec:
         app: data-repo
         component: data-repo-oidc-proxy
     spec:
+      serviceAccountName: jade-sa
+      volumes:
+      - name: server-cert
+        secret:
+          secretName: server-cert
+      - name: server-key
+        secret:
+          secretName: server-key
       containers:
       - name: oidc-proxy-container
-        image: broadinstitute/openidc-proxy:db_ldap_2_3_3
+        image: broadinstitute/openidc-proxy:bernick_tcell
```

Comment:
That's pretty specific! Since this is YAML, maybe you can drop in some documentation to explain.

Reply:
Will do -- this is the recommendation of the security team.